tools/clippy/BUILD

```diff
@@ -0,0 +1 @@
+exports_files(["clippy.toml"])
```

Comment:
Can you rename this file `BUILD` file to `BUILD.bazel` for consistency with the rest of the repo?

Reply:
Renamed